index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/main.js" defer="defer">
```

Comment:
don't need to split `<script>` tag by empty space. should be: `<script src="js/main.js" defer="defer"></script>`